<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminlogin</class>
 <widget class="QMainWindow" name="adminlogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Admin Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adminUser">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adminPass">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="adminloginBttn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBttn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>365</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBttn</sender>
   <signal>clicked()</signal>
   <receiver>adminlogin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_exitBttn_clicked()</slot>
  <slot>on_adminloginBttn_clicked()</slot>
 </slots>
</ui>
